sql/moz-fx-data-shared-prod/static/iana_tls_cipher_suites/schema.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "mode": "NULLABLE",
+    "name": "Value",
```

Comment:
Can we make these field names all lower_snake_case? We're skipping the header row in the file, so I don't think there's any reason to be restricted to the names in the CSV itself.

I also think it's worth adding descriptions to each of these fields here.